csharp/tutorials/AsyncStreams/finished/IssuePRreport/IssuePRreport/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GitHubActivityReport
+{
+    public class GraphQLRequest
+    {
+        public string query { get; set; }
```

Comment:
Nit: Use Pascal case for public members.

Reply:
That also means adding the `JsonPropertyAttribute` on the declarations to match the expected names for the GraphQL request.

Reply:
That's what I thought.